integration-tests/eks/resourceCount_linuxonly_test.go

```diff
@@ -0,0 +1,16 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+//go:build linuxonly
+// +build linuxonly
```

Comment:
Can we not use `GOOS` tags?
Even if the tags don't work lets do `linux` instead of `linuxonly`, same for windows
https://www.digitalocean.com/community/tutorials/building-go-applications-for-different-operating-systems-and-architectures#using-goos-and-goarch-filename-suffixes

Errr maybe I don't understand this.  What is the point of `windowslinux`? how would that be different than just `resourceCount_test.go`?

Reply:
These tests file will always run on Linux so adding GOOS tags wont work. 
We are adding linuxonly and windowslinux to differentiate running these tests against linux only cluster and mixed clusters